pkg/build/metrics/prometheus/metrics.go

```diff
@@ -93,55 +81,60 @@ func (bc *buildCollector) Collect(ch chan<- prometheus.Metric) {
 
 	// since we do not collect terminal build metrics on a per build basis, collectBuild will return counts
 	// to be added to the total amount posted to prometheus
-	var cancelledCount, completeCount int
+	var cancelledCount, completeCount, newCount, pendingCount, runningCount int
 	reasons := map[string]int{}
 	for _, b := range result {
-		cc, cp, r := bc.collectBuild(ch, b)
+		cc, cp, nc, pc, rc, r := bc.collectBuild(ch, b)
 		for key, value := range r {
 			reasons[key] = reasons[key] + value
 		}
 		cancelledCount = cancelledCount + cc
 		completeCount = completeCount + cp
+		newCount = newCount + nc
+		pendingCount = pendingCount + pc
+		runningCount = runningCount + rc
 	}
 	// explicitly note there are no failed builds
```

Comment:
hm.  should we not be doing this for all the phases?

Reply:
failed / error are handled by the reason map

Reply:
will add new/pending/running

Reply:
ah, ok i think i understand now.  nothing else sets the gauge if there are no builds with a phase=failed because there are no "reasons" to iterate.

Reply:
(I was confused about why we had to explicitly set that one to zero, instead of just setting it to whatever the count was).

but yes, need to add addCountGauge calls for the new + pending count values.  I missed that :)